src/apiClient.ts

```diff
@@ -19,12 +19,10 @@ export default class ApiClient<TResource> {
         try {
             const response = await this.adapter.execute(this.options);
             this.handleSuccess(response);
-        }
-        catch (error: unknown) {
+        } catch (error: unknown) {
             if (error instanceof AdapterError) {
                 this.handleError(error);
-            } 
-            else if (error instanceof Error) {
+            } else if (error instanceof Error) {
```

Comment:
I know this was just a formatting change, but this code block is not safe.  An error in JS is not always an instance of an error object. In JS anything can be thrown, a string, number. We should handle the case when it's not an instance error object too, or we could potentially be swallowing errors.